devops/README.md

```diff
@@ -4,14 +4,35 @@ The `arangodb.yml` Ansible playbook in this directory installs and configures
 ArangoDB on an Ubuntu server. It does the following:
 - installs ArangoDB
 - sets a password on the root account
-- configures it to listen on all interfaces
+- copies an ssl certificate
+- configures it to listen on all interfaces using ssl
 - starts the service
 
 To set the root account password, you can either supply it on the command line
 (see below) or just let Ansible prompt you for it.
 
-To run it via ssh, run a command such as the following:
+To use a wildcard ssl certificate for your domain you'll need to supply one, we use Let's Encrypt, and pass in the file path to the command.
+If you're using a provider other than Let's Encrypt, you'll have to modify the playbook to work with your provider and file structure. 
+If you're using Let's Encrypt, follow along with [this](https://medium.com/@saurabh6790/generate-wildcard-ssl-certificate-using-lets-encrypt-certbot-273e432794d7) 
+guide to generate a wildcard certificate. This playbook assumes you have the certificate files and that they're in the default location.
+If you don't want a ssl certificate, you'll have to modify the playbook and remove the `ssl_cert_path` env var from the command below.
+
+To run the ansible playbook via ssh, run a command such as the following:
+
+```
+sudo ansible-playbook arangodb.yml -i <target-hostname>, --ssh-extra-args="-i <identity.pem>" -e arangodb_root_password=<password> ssl_cert_path=<path-to-files>
+```
+
+*NOTE*: You have to use sudo so that you may copy the local, write protected certificate files.
```

Comment:
That's true. The problem is that the private key file is only readable to root after running certbot. I was concerned that telling people to change the permissions of the files on their machines would be less secure than just telling them to run ansible as root. I could also add some documentation that changes the private key file permission to readable and back to unreadable if that makes more sense?

Reply:
If you think there will be use cases where we'll want to use this without a SSL cert, I can add the conditionals. For people installing it on their own machines for testing they should use the `docker-compose` method, and if they're deploying to a webserver they should probably be using SSL, right?

Reply:
> That's true. The problem is that the private key file is only readable to root after running certbot. I was concerned that telling people to change the permissions of the files on their machines would be less secure than just telling them to run ansible as root. I could also add some documentation that changes the private key file permission to readable and back to unreadable if that makes more sense?

People should only run as root when it's required, and because you can use sudo to change permissions momentarily on a single file, you should do that instead of running the entire provisioning tool as root. I think you can just explain that changing permissions on the file may be needed to make it readable for `ansible-playbook` to copy it, and it's not even necessary to explain step-by-step how to do it. We need to assume some level of familiarity with Linux, or else our docs will grow exponentially.

But we definitely should *not* instruct people to run `ansible-playbook` as root. I don't know everything it does, and for that reason running it as root is dangerous.

Reply:
> If you think there will be use cases where we'll want to use this without a SSL cert, I can add the conditionals. For people installing it on their own machines for testing they should use the `docker-compose` method, and if they're deploying to a webserver they should probably be using SSL, right?

Well, the main use case right now is for us to test the playbook... with an SSL cert. So you're right, there's no need to do more work now to make it work for non-SSL cert use cases.

Reply:
Added a note about local file permissions in the README. Removed `sudo` from the commands in b594996